Register service.alpha.kubernetes.io/tolerate-unready-endpoints Ann… (#39122)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/_index.md

- `onstart`: The APIService should be reconciled when an API server starts up, but not otherwise.
- `true`: The API server should reconcile this APIService continuously.

### service.alpha.kubernetes.io/tolerate-unready-endpoints (deprecated)

Used on: StatefulSet

> This annotation on the Service is denoting if the endpoints controller should go ahead and create endpoints for unready pods.

Reviewer: ```suggestion
This annotation on a Service denotes if the Endpoints controller should go ahead and create Endpoints for unready Pods.
```

Author: Done

> Endpoints of these Services retain their DNS records and continue receiving traffic for the Service from the moment the kubelet starts all containers in the pod and marks it "Running", till the kubelet stops all containers and deletes the pod from the apiserver.

Reviewer: Please wrap the long lines as we did for other items.

Author: @tengqm i have Addressed the changes . PTAL!!

- `onstart`: The APIService should be reconciled when an API server starts up, but not otherwise.
- `true`: The API server should reconcile this APIService continuously.

### service.alpha.kubernetes.io/tolerate-unready-endpoints (deprecated)

Used on: StatefulSet

This annotation on a Service denotes if the Endpoints controller should go ahead and create Endpoints for unready Pods.
Endpoints of these Services retain their DNS records and continue receiving
traffic for the Service from the moment the kubelet starts all containers in the pod

> and marks it "Running", till the kubelet stops all containers and deletes the pod from

Reviewer 2: ```suggestion
and marks it _Running_, til the kubelet stops all containers and deletes the pod from
```

Author: Addressed the changes.

> the apiserver.

Reviewer: ```suggestion
the API server.
```

Author: Done
